src/preload/preload.ts

```diff
@@ -138,6 +149,8 @@ export async function setupFiddleGlobal() {
     },
     getTestTemplate: () => ipcRenderer.invoke(IpcEvents.GET_TEST_TEMPLATE),
     getUsername: () => ipcRenderer.sendSync(IpcEvents.GET_USERNAME),
+    getVersionState: (version: string) =>
+      ipcRenderer.sendSync(IpcEvents.GET_VERSION_STATE, version),
```

Comment:
Can this be asynchronous? I realize we have a bunch of synchronous calls here, but IMO we should avoid them in new code if it's not too much of a hassle, they can add up and come back to bite us 😬

Reply:
This was made sync to match the existing code without needing to do more extensive refactoring. Particularly this is used in a constructor where we can't use `async` calls. It can probably be updated to work with asynchronously populating the version states, but that will require more extensive testing so I don't want to touch that in this PR.

https://github.com/electron/fiddle/blob/74bde543f3cbb2818e9b0ab020a0e880d455b949/src/renderer/state.ts#L355-L362

I agree with the overall sentiment though - once this all lands we should look at doing future refactors to change any usages of `ipcRenderer.sendSync` that we can.